code/index.html

```diff
@@ -1,30 +1,167 @@
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
+     <!-- video or image as a header is cool :) -->
+    <header class="header">
+      <div class="hero">
+        <video width="" height="" playsinline autoplay muted loop poster="" class="bgvideo">
+          <source src="pageturner.mp4" type="video/mp4">
+        </video>
+      </div>
+      
+      <div class="logo">
+        <img class="imglogo" src="BC_logoV2.png" alt="logotype for Bookbinders Club" />
+      </div>
 
-    <!-- Signup form -->
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      
+      <nav class="nav">
+        <a class="nav-item" href="#" >About</a>
+        <a class="nav-item" href="#">Courses</a>
+        <a class="nav-item" href="#">Merchandise</a>
+        <a class="nav-item" href="#">Contact</a>
+      </nav>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+      <section class="headline">
+        <h1>A Forgotten Craftsmanship</h1>
+        <p>Learn Bookbinding Today!</p>
+      </section>
+    </header>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+    <div class="main">
+
+      <!-- Signup form -->
+      <section class="form">
+        <form action="http://httpbin.org/anything" method="POST">
+        <h2>Sign up here!</h2>
+        <h5>To get more information and notification alernatives.</h5>
+     
+        <div class= "text-input">
+          <label class="name">
+            Name<br>
+            <input class="textfield" type="text" name="message" required></input><br>
+          </label>
+          <label class="e-mail">
+            E-mail<br>
+            <input class="textfield" type="email" name="email" required><br>
+          </label>
+        </div>
+
+        <div class="select-menu-input">
+          <label>
+            Course Location:<br>
+            <select name="location" id=""><br>
+              <option value="Stockholm">Stockholm</option><br>
+              <option value="Gothenburg">Gothenburg</option><br>
+              <option value="Malmo">Malmo</option><br>
+            </select>
+          </label><br>
+        </div>
+
+        <div class="radio-input">
+          <label>
+            What type of information would you like to recieve?<br>
+              <input type="radio" name="WeeklyNewsletter">
+              Weekly Newsletter
+          </label>
+
+          <label><br>
+            <input type="radio" name="Courseinfo">
+            Course Information
+          </label>
+
+          <label><br>
+            <input type="radio" name="ApplicationPapers">
+            Application Papers
+          </label>
+
+          <label><br>
+            <input type="radio" name="MonthlyNewsletter">
+            Monthly Newsletter
+          </label>
+        </div>
```

Comment:
If the user shoul be able to check more than one option, these should be checkboxes. If the user should only be able to check one, you should make sure to have the same `name` on all of the radio buttons, but different values. Read more in week 3's general feedback document